LeanMapperQuery/Query.php

```diff
@@ -419,6 +433,13 @@ protected function getFluent()
 
 	////////////////////////////////////////////////////
 
+	public function cast($entityClass = NULL)
+	{
+		$this->entityClass = $entityClass;
```

Comment:
Chceme umožnit vícenásobné volání téhle metody?

Reply:
Když vytváříš query object sám, tak je ti to asi jedno, ale když si ho necháš vytvořit pomocí nějaké továrny (třeba `$query = $invoiceFacade->queryInvoices()`), může být vícenásobné volání užitečné (když by `cast()` volala už továrna a ty bys to pak potřeboval přetížit).

Ale popravdě řečeno nevím, co bude lepší - oboje má něco do sebe. Možná by se mohlo teď vícenásobné volání zakázat a když by se to ukázalo jako problém, tak by se to omezení zrušilo.

Reply:
A možná dát té metodě `cast` i nějaký doc comment?